lib/friendly_id/mobility.rb

```diff
@@ -23,15 +23,16 @@ def included(model_class)
 
         mod = Module.new do
           def friendly
-            super.send(::Mobility.query_method)
+            # TODO: Make this constant public in Mobility 1.1 so we don't need const_get
+            super.extending(::Mobility::Plugins::ActiveRecord::Query.const_get(:QueryExtension))
```

Comment:
Working on some changes to make more constants public from the query plugin: https://github.com/shioyama/mobility/pull/471

I'll probably release shortly in a 1.1. Didn't want to include it in 1.0 to delay that release any longer. For now this should do.